File: app/src/main/java/com/haishinkit/util/AndroidLogAdapter.java
package com.haishinkit.util;

import android.util.Log;

public final class AndroidLogAdapter implements ILogAdapter {
    public int v(String tag, String msg) {
        return Log.v(tag, msg);
    }

    public int v(String tag, String msg, Throwable tr) {
        return Log.v(tag, msg, tr);
    }

    public int d(String tag, String msg) {
        return Log.d(tag, msg);
    }

    public int d(String tag, String msg, Throwable tr) {
        return Log.d(tag, msg, tr);
    }

    public int i(String tag, String msg) {
        return Log.i(tag, msg);
    }

    public int i(String tag, String msg, Throwable tr) {
        return Log.i(tag, msg, tr);
    }

    public int w(String tag, String msg) {
        return Log.w(tag, msg);
    }

    public int w(String tag, String msg, Throwable tr) {
        return Log.w(tag, msg, tr);
    }

    public int w(String tag, Throwable tr) {
        return Log.w(tag, tr);
    }

    public int e(String tag, String msg) {
        return Log.e(tag, msg);
    }

    public int e(String tag, String msg, Throwable tr) {
        return Log.e(tag, msg, tr);
    }
}

File: haishinkit/src/main/java/com/haishinkit/util/AndroidLogAdapter.kt
package com.haishinkit.util

import android.util.Log

class AndroidLogAdapter : ILogAdapter {
    override fun v(tag: String, msg: String): Int {
        return Log.v(tag, msg)
    }

    override fun v(tag: String, msg: String, tr: Throwable): Int {
        return Log.v(tag, msg, tr)
    }

    override fun d(tag: String, msg: String): Int {
        return Log.d(tag, msg)
    }

    override fun d(tag: String, msg: String, tr: Throwable): Int {
        return Log.d(tag, msg, tr)
    }

    override fun i(tag: String, msg: String): Int {
        return Log.i(tag, msg)
    }

    override fun i(tag: String, msg: String, tr: Throwable): Int {
        return Log.i(tag, msg, tr)
    }

    override fun w(tag: String, msg: String): Int {
        return Log.w(tag, msg)
    }

    override fun w(tag: String, msg: String, tr: Throwable): Int {
        return Log.w(tag, msg, tr)
    }

    override fun w(tag: String, tr: Throwable): Int {
        return Log.w(tag, tr)
    }

    override fun e(tag: String, msg: String): Int {
        return Log.e(tag, msg)
    }

    override fun e(tag: String, msg: String, tr: Throwable): Int {
        return Log.e(tag, msg, tr)
    }
}
